src/views/ItemPage.vue

```diff
@@ -121,23 +122,21 @@ watch(
   </div>
 </template>
 
-<style>
+<style scoped>
```

Comment:
@hollote, I added scoped in styles, but some styles doesn't work(.p-inputtext, .p-menuitem-link, .close-button > span). What should I do?

Reply:
check `::deep` for vue styles

Reply:
done